### PR TITLE
added local tuple variable

### PR DESCRIPTION
**Pull request by lennessyy**

### CODE REVIEW

**Comment by lennessyy** [comment]
Hi @jue-yuan , this is the draft for the local tuple variable feature. Please take look and let me know if I missed anything.

File: modules/querying/pages/declaration-and-assignment-statements.adoc [hunk 1]
```diff
@@ -163,7 +165,7 @@ CREATE QUERY baseTypeVariable() FOR GRAPH socialNet {
 Base type variables declared in a DML-sub statement, such as in a statement inside a `ACCUM`, `POST-ACCUM`, or `UPDATE SET` clause, are called _local_ _base type variables_.
 
 Local base type variables are block-scoped and are accessible in the block where they are declared only.
-Within a local base type variable's scope, you cannot another local base type variable with the same name at the same level.
+Within a local base type variable's scope, you cannot another local base type variable, local container variable, or local tuple variable with the same name at the same level.
```
Comment (jue-yuan) [inline modules/querying/pages/declaration-and-assignment-statements.adoc:168]: Missing a verb between `cannot` and `another`

File: modules/querying/pages/declaration-and-assignment-statements.adoc [hunk 2]
```diff
@@ -280,7 +282,7 @@ SET<INT> set1 = (1, 2, 3) <1>
 ====
 
 Local container variables are block-scoped and are accessible in the block where they are declared only.
-Within a local container variable’s scope, you cannot another local container variable or local base type variable with the same name at the same level.
+Within a local container variable’s scope, you cannot another local container variable, local tuple variable, or local base type variable with the same name at the same level.
```
Comment (jue-yuan) [inline modules/querying/pages/declaration-and-assignment-statements.adoc:285]: Same as above.

File: modules/querying/pages/declaration-and-assignment-statements.adoc [hunk 3]
```diff
@@ -361,6 +363,49 @@ returns(SetAccum<tuple<int, string>>){
 --
 ====
 
+=== Local tuple variable
+Variables declared inside a DML-block storing tuple values are called _local tuple variables_.
+The value for a local tuple variable is assigned at declaration and cannot be updated afterwards.
```
Comment (jue-yuan) [inline modules/querying/pages/declaration-and-assignment-statements.adoc:368]: Local tuple can be updated by an assignment as well.

**Comment by lennessyy** [comment]
@jue-yuan Thanks for proofreading. I fixed the errors - I misunderstood when I read that they can only be assigned. I thought it meant "Can only be assigned but cannot be updated", but now I see it means "can only be assigned and cannot accumulate"

**Comment by jue-yuan** [comment]
> @jue-yuan Thanks for proofreading. I fixed the errors - I misunderstood when I read that they can only be assigned. I thought it meant "Can only be assigned but cannot be updated", but now I see it means "can only be assigned and cannot accumulate"

Yes, that's because it is not a container.

**Review by jue-yuan — APPROVED** [review]

**Pull request merged**